packages/intent-filler/src/config/chains.toml

```diff
@@ -0,0 +1,44 @@
+[chains.bsc-chapel]
+type = "evm"
+chainId = 97
```

Comment:
probably don't need this, we can get chainId from state machine Id.

Reply:
I am using these inside chainConfig service.

Reply:
Yeah, you can extract from the state machine is when parsing the toml file

Reply:
Done!